src/tree_schema.h

```diff
@@ -72,7 +72,7 @@ struct lyxp_expr;
  * the default nodes are not added into any data tree and mandatory nodes are not checked in the data trees.
  *
  * The compiled schema tree nodes are able to hold private objects (::lysc_node.priv as a pointer to a structure,
- * function, variable, ...) used by a caller application.
+ * function, variable, ...) used by a caller application unless ::LY_CTX_SET_PRIV_PARSED is not set.
```

Comment:
Should be `unless ::LY_CTX_SET_PRIV_PARSED is set`. But you can be more verbose here and at least briefly explain that if the option is set, the `priv` objects are actually filled by *libyang*.

Reply:
ok